vizro-core/tests/unit/vizro/models/_navigation/test_accordion.py

```diff
@@ -13,41 +14,39 @@
 class TestAccordionInstantiation:
     """Tests accordion model instantiation."""
 
-    def test_create_accordion_default(self):
-        accordion = Accordion(id="accordion_id")
-        assert accordion.id == "accordion_id"
-        assert accordion.pages == ["Page 1", "Page 2"]
-
-    def test_create_accordion_pages_as_list(self, pages_as_list):
+    def test_accordion_valid_pages_as_list(self, pages_as_list):
         accordion = Accordion(pages=pages_as_list, id="accordion_id")
         assert accordion.id == "accordion_id"
-        assert accordion.pages == pages_as_list
+        assert accordion.pages == {"SELECT PAGE": pages_as_list}
 
-    def test_create_accordion_pages_as_dict(self, pages_as_dict):
+    def test_accordion_valid_pages_as_dict(self, pages_as_dict):
         accordion = Accordion(pages=pages_as_dict, id="accordion_id")
         assert accordion.id == "accordion_id"
         assert accordion.pages == pages_as_dict
 
-    def test_field_invalid_pages_input_type(self):
-        with pytest.raises(ValidationError, match="2 validation errors for Accordion"):
-            Accordion(pages=[vm.Button()])
+    def test_navigation_valid_pages_not_all_included(self):
+        accordion = Accordion(pages=["Page 1"], id="accordion_id")
+        assert accordion.id == "accordion_id"
+        assert accordion.pages == {"SELECT PAGE": ["Page 1"]}
+
+    def test_invalid_field_pages_required(self):
+        with pytest.raises(ValidationError, match="field required"):
+            Accordion()
 
-    def test_field_invalid_pages_empty_list(self):
+    @pytest.mark.parametrize("pages", [{"SELECT PAGE": []}, []])
+    def test_invalid_field_pages_no_ids_provided(self, pages):
         with pytest.raises(ValidationError, match="Ensure this value has at least 1 item."):
-            Accordion(pages=[])
+            Accordion(pages=pages)
+
+    def test_invalid_field_pages_wrong_input_type(self):
```

Comment:
Random comment - let's not change anything here, but worth bearing in mind.

To stop people from needing to copy and paste page IDs we should note this pattern is a good one:
```
pages = [Page(...), Page(...)]
dashboard = Dashboard(pages=pages, navigation=Navigation(pages=[page.id for page in pages])
```

If people find this confusing and make the mistake of inputting the `Page` object itself a lot then potentially we could also enable all these `pages` fields to accept that and immediately extract the `id` from that, so the above would be even simpler:
```
pages = [Page(...), Page(...)]
dashboard = Dashboard(pages=pages, navigation=Navigation(pages=pages))
# somewhere in the validator we do navigation.pages = [page.id for page in pages]
```
...but no need to do that unless it's a proven pain point.

Reply:
I remember there was a warning for "forgotten" pages before, why did we decide to get rid of it? Is it because you mentioned above that somethins is now autmatically added?

Reply:
Yes, agree! I added the test in here because I think at some point it got confusing with the other `Navigation` selectors and all of them having a `pages` argument. As long as it's clear that dashboard.pages accept Page object and all Navigation related models Page IDs, I think it's fine!

Reply:
@maxschulz-COL - we decided that the effort to maintain the warning wasn't worth the added benefit. As far as I understood @nadijagraca it became increasingly difficult to do that check with the Icons/Accordions. 

The user will still get a visual indication whether they included all pages or not, as only the included Pages in the Navigation will be visible